dlt/common/storages/load_storage.py

```diff
@@ -87,6 +87,7 @@ class LoadPackageInfo(NamedTuple):
     package_path: str
     state: TLoadPackageState
     schema_name: str
+    schema: Schema
```

Comment:
fyi i am adding the full schema to the load info here

Reply:
IMO way better if you add TypedDict with schema content, not the object itself. this is being pickled and dumped into trace so obviously dicts work better